README.md

```diff
@@ -16,6 +16,20 @@ It uses [Controllers](https://kubernetes.io/docs/concepts/architecture/controlle
 
 This project is scaffolded using [Kubebuilder](https://book.kubebuilder.io), and all the Kubebuilder [`makefile` helpers](https://book.kubebuilder.io/reference/makefile-helpers.html) can be used.
 
+## Install
+
+Run the following command to install the latest version of the NATS manager on your cluster:
```

Comment:
```suggestion
1. To install the latest version of the NATS manager on your cluster, run:
```
"First purpose, then instruction" makes it easier to detect what this is about (especially in contrast to the 2nd instruction).